packages/functions-runtime/src/flows/index.ts

```diff
@@ -39,14 +39,15 @@ const defaultOpts = {
   timeout: 60000,
 };
 
-export interface FlowContext<C extends FlowConfig, E = any, S = any> {
+export interface FlowContext<C extends FlowConfig, E = any, S = any, Id = any> {
```

Comment:
I think we can do this because these types will and must always be provided

```suggestion
export interface FlowContext<C extends FlowConfig, E, S, Id> {
```